### ISSUE TITLE
When in mobile view, two menu icons display

### ISSUE THREAD

**Issue by lnhsingh** [cockroachdb/docs#9657]
From John S: here’s some usability feedback about how the docs look when not full screen. If you’re wondering why I have the docs not full screen, I like to have them up on 1/2 of my screen while I’m writing SQL or working on the terminal on the other 1/2. But when the docs get about 2/3rd of the screen, they snap into the mobile responsive layout, and this happens to the navigation menu:

![image](https://user-images.githubusercontent.com/15386648/106334299-58b78b80-6258-11eb-8a47-0915f1d04fe1.png)

**Comment by jseldess** [comment]
@luciaaaaaaa, needs design input.

**Comment by jseldess** [comment]
I'm not sure we'll be able to get around this unless we hide the menu for the topnav in mobile view. Thoughts, @luciaaaaaaa?

**Comment by luciaaaaaaa** [comment]
I agree, we could hide the top nav as the users scroll down?
Just checked how it looks now and it looks way better 
![image](https://user-images.githubusercontent.com/48258790/112525996-d6d77080-8d77-11eb-8102-7efa5df4f57d.png)

@lnhsingh what do you think, should we still try make the Docs dropdown more prominent?

**Comment by lnhsingh** [comment]
Agreed, we can hide top nav in mobile view. I don't think we need to make
the Docs dropdown more prominent otherwise
